x/protorev/keeper/rebalance.go

```diff
@@ -378,9 +378,9 @@ func (k Keeper) ExecuteTrade(ctx sdk.Context, route poolmanagertypes.SwapAmountI
 		return err
 	}
 
-	// Send the developer fee to the developer address
-	if err := k.SendDeveloperFee(ctx, sdk.NewCoin(inputCoin.Denom, profit)); err != nil {
-		ctx.Logger().Error("failed to send developer fee: " + err.Error())
+	// Distribute the arbitrage profit.
+	if err := k.DistributeProfit(ctx, sdk.NewCoin(inputCoin.Denom, profit)); err != nil {
+		ctx.Logger().Error("failed to distribute arbitrage profit: " + err.Error())
```

Comment:
curious since this was here before hand, should this instead error out?

Reply:
hrmm, I think its ok if the arb profit fails to distr